string label
uint16 track_id
sensor_msgs/Image mask
Keypoint[] kpts_curr
Keypoint[] kpts_last
